packages/rancher-windows-gmsa/charts/questions.yaml

```diff
@@ -0,0 +1,59 @@
+questions:
+  - variable: credential.enabled
+    default: true
+    description: Create a GMSA Credential Spec when installing GMSA Webhook
+    label: Create a GMSA Credential Spec
+    type: boolean
+    group: "Credential Spec"
+    show_subquestion_if: true
+    subquestions:
+      - variable: credential.domainJoinConfig.machineAccountName
+        label: GMSA Account Name
+        description: Username of the GMSA account
+        type: string
+        required: true
+      - variable: credential.domainJoinConfig.guid
+        label: GUID
+        description: GUID of the Service Account
+        type: string
+        required: true
+      - variable: credential.domainJoinConfig.sid
+        label: SID
+        description: SID of the GMSA Account
+        type: string
+        required: true
+      - variable: credential.domainJoinConfig.dnsName
+        label: DNS Domain Name
+        description: Name of the domain in DNS
+        type: string
+        required: true
+      - variable: credential.domainJoinConfig.dnsTreeName
+        label: DNS Tree Domain
+        description: Root name of the domain in DNS
+        type: string
+        required: true
+      - variable: credential.domainJoinConfig.netBiosName
+        label: NETBIOS Name
+        description: NETBIOS Name for the domain.
+        type: string
+        required: true
+
+  - variable: certificates.certManager.enabled
+    default: true
+    description: Enable cert manager integration. Cert manager should be already installed
+    label: Enable Cert Manager integration
+    type: boolean
+    group: "Certificates"
+    show_subquestion_if: false
+    subquestions:
+      - variable: certificates.secretName
+        default: webhook-server-cert
+        description: Use certificates from secret. Secret should exists in the app namespace, with certs data (ca.crt, tls.crt & tls.key)
+        label: Secret name
+        type: string
+        required: true
+      - variable: certificates.caBundle
+        description: Use self signed CA Bundle. It should be provided in base64 format

```

Comment:
```suggestion
        description: Provide self-signed CA Bundle as a base64-encoded string
```

Reply:
I'm also concerned whether we should allow users to pass in a caBundle this way, since it's a security risk right? Anyone with permissions to view the values provided to the Helm deployment can `base64 -d` it and grab the underlying certs used.

I think the existing two options of a) use generated or b) put it in a secret are sufficient and this can be dropped as a Helm chart option. What do you think?

Reply:
This is how this is handled in the ExternalIP chart.